docs/categorical-histogram-dap-04.md

```diff
@@ -0,0 +1,113 @@
+---
+slug: /categorical-histogram-dap-04
+sidebar_position: 2
+---
+
+# Emulating Categorical Histograms with DAP draft-04
+
+The Prio3Histogram VDAF has changed between drafts 04 and 07 of DAP. Initially,
+it provided the functionality of a conventional histogram of a continuous
+variable. As of draft-07, its API has changed to be more flexible, supporting
+other use cases like categorical variables, multi-dimensional histograms, etc.
```

Comment:
```suggestion
other use cases like categorical variables, multidimensional histograms, etc.
```

In section below, this is not hyphenated. I have no idea if it should be hyphenated in both places or neither